src/lib/libopenal.js

```diff
@@ -3778,7 +3778,7 @@ var LibraryOpenAL = {
         break;
       default:
 #if OPENAL_DEBUG
-        dbg(`alBufferData() called with invalid format ${format}`;
+        dbg(`alBufferData() called with invalid format ${format}`);
```

Comment:
This change and the one below are fixes for syntax errors.  Can you split these out too?